tests/test_converter.py

```diff
@@ -716,39 +716,40 @@ def test_map_wires_instantiate_quantum_circuit_with_registers(self, recorder):
         qr2 = QuantumRegister(1)
         qr3 = QuantumRegister(1)
         qc = QuantumCircuit(qr1, qr2, qr3)
-        qc_wires = [(q.register.name, q.index) for q in qc.qubits]
+        qc_wires = [hash(q) for q in qc.qubits]
 
         mapped_wires = map_wires(qc_wires, wires)
 
         assert len(mapped_wires) == len(wires)
         assert list(mapped_wires.keys()) == wires
         for q in qc.qubits:
-            assert (q.register.name, q.index) in mapped_wires.values()
+            assert hash(q) in mapped_wires.values()
 
     def test_map_wires_provided_non_standard_order(self, recorder):
         """Tests the map_wires function for wires of non-standard order."""
 
         wires = [1, 2, 0]
         qc = QuantumCircuit(3)
-        qc_wires = [(q.register.name, q.index) for q in qc.qubits]
+        qc_wires = [hash(q) for q in qc.qubits]
 
         mapped_wires = map_wires(qc_wires, wires)
 
+        for q in qc.qubits:
```

Comment:
Any reason why these asserts were placed before lines 740-741? (they came after these lines originally)

Reply:
just keeping single line asserts together and not interrupted. Looks nicer?